openedx/core/djangoapps/user_api/accounts/helpers.py

```diff
@@ -13,45 +13,81 @@
     'small': 30
 }
 _PROFILE_IMAGE_SIZES = PROFILE_IMAGE_SIZES_MAP.values()
-PROFILE_IMAGE_FORMAT = 'jpg'
 
 
-def get_profile_image_url_for_user(user, size):
-    """Return the URL to a user's profile image for a given size.
-    Note that based on the value of
-    django.conf.settings.PROFILE_IMAGE_DOMAIN, the URL may be relative,
-    and in that case the caller is responsible for constructing the full
-    URL.
+def get_profile_image_storage():
+    """
+    Configures and returns a django Storage instance that can be used
+    to physically locate, read and write profile images.
+    """
+    storage_class = get_storage_class(settings.PROFILE_IMAGE_BACKEND)
+    return storage_class(base_url=(settings.PROFILE_IMAGE_DOMAIN + settings.PROFILE_IMAGE_URL_PATH))
+
+
+def _make_name(username):
```

Comment:
Since this is only called in one place, might as just move the code there (it will address one of @dan-f's comments).